Update italian translation

ad-daniel opened the pull request:
**Description**
Continuation of the italian translation of Webots.

stefaniapedrazzi commented on resources/translations/wb_it.ts:
```diff
@@ -1713,11 +1719,11 @@ Restore initial state of the simulation. (%1+Shift+F)</source>
     <message>
         <source>Could not create a symbolic link of dynamic library: &apos;%1&apos;.</source>
-        <translation type="unfinished"></translation>
+        <translation>Impossibile creare link simbolico della libreria dinamica: &apos;%1&apos;.</translation>
```
```suggestion
        <translation>Impossibile creare un link simbolico della libreria dinamica: &apos;%1&apos;.</translation>
```

stefaniapedrazzi commented on resources/translations/wb_it.ts:
```diff
@@ -1668,41 +1672,43 @@ Restore initial state of the simulation. (%1+Shift+F)</source>
         <source>HTML5 animation successfully exported in &apos;%1&apos;
 </source>
-        <translation type="unfinished"></translation>
+        <translation>animazione HTML5 esportata con successo in &apos;%1&apos;
```
```suggestion
        <translation>Animazione HTML5 esportata con successo in &apos;%1&apos;
```

stefaniapedrazzi commented on resources/translations/wb_it.ts:
```diff
@@ -1668,41 +1672,43 @@ Restore initial state of the simulation. (%1+Shift+F)</source>
     <message>
         <source>HTML5 export failed</source>
-        <translation type="unfinished"></translation>
+        <translation>esportazione HTML5 fallita</translation>
```
```suggestion
        <translation>Esportazione HTML5 fallita</translation>
```

stefaniapedrazzi commented on resources/translations/wb_it.ts:
```diff
@@ -1668,41 +1672,43 @@ Restore initial state of the simulation. (%1+Shift+F)</source>
     <message>
         <source>Error when stopping the HTML5 animation recording: &apos;%1&apos;</source>
-        <translation type="unfinished"></translation>
+        <translation>Errore nel interrompere la registrazione dell&apos;animazione HTML5: &apos;%1&apos;</translation>
```
```suggestion
        <translation>Errore nell&apos;interrompere la registrazione dell&apos;animazione HTML5: &apos;%1&apos;</translation>
```

stefaniapedrazzi commented on resources/translations/wb_it.ts:
```diff
@@ -1629,31 +1633,31 @@ Restore initial state of the simulation. (%1+Shift+F)</source>
     <message>
         <source>PROTO nodes (Extra Projects)</source>
-        <translation type="unfinished"></translation>
+        <translation>nodi PROTO (Progetti Aggiuntivi)</translation>
```
```suggestion
        <translation>Nodi PROTO (Progetti Aggiuntivi)</translation>
```

stefaniapedrazzi commented on resources/translations/wb_it.ts:
```diff
@@ -1629,31 +1633,31 @@ Restore initial state of the simulation. (%1+Shift+F)</source>
     <message>
         <source>PROTO nodes (Current Project)</source>
-        <translation type="unfinished"></translation>
+        <translation>nodi PROTO (Progetto Corrente)</translation>
```
```suggestion
        <translation>Nodi PROTO (Progetto Corrente)</translation>
```

stefaniapedrazzi commented on resources/translations/wb_it.ts:
```diff
@@ -1589,35 +1593,35 @@ Restore initial state of the simulation. (%1+Shift+F)</source>
     <message>
         <source>This folder lists all suitable PROTO nodes from the local &apos;protos&apos; directory: &apos;%1&apos;.</source>
-        <translation type="unfinished"></translation>
+        <translation>Questa cartella elenca tutti i nodi PROTO idonei della directory locale &apos;protos&apos;: &apos;%1&apos;.</translation>
```
```suggestion
        <translation>Questa sezione elenca tutti i nodi PROTO idonei della cartella locale &apos;protos&apos;: &apos;%1&apos;.</translation>
```

stefaniapedrazzi commented on resources/translations/wb_it.ts:
```diff
@@ -1589,35 +1593,35 @@ Restore initial state of the simulation. (%1+Shift+F)</source>
     <message>
         <source>This folder lists all suitable PROTO nodes provided by Webots.</source>
-        <translation type="unfinished"></translation>
+        <translation>Questa cartella elenca tutti i nodi PROTO idonei forniti da Webots.</translation>
```
```suggestion
        <translation>Questa sezione elenca tutti i nodi PROTO idonei forniti da Webots.</translation>
```

stefaniapedrazzi commented on resources/translations/wb_it.ts:
```diff
@@ -1513,28 +1516,29 @@ Restore initial state of the simulation. (%1+Shift+F)</source>
+        <translation>Aggiungi</translation>
     </message>
     <message>
-        <source>Insert the identity matrix 
```
The source text should not be modified

stefaniapedrazzi commented on resources/translations/wb_it.ts:
```diff
@@ -1326,7 +1329,7 @@ Restore initial state of the simulation. (%1+Shift+F)</source>
     <message>
         <source>Restore &amp;Viewpoint</source>
-        <translation type="unfinished"></translation>
+        <translation>Restora &amp;Punto di Vista</translation>
```
```suggestion
        <translation>Ripristina &amp;Punto di Vista</translation>
```

stefaniapedrazzi commented on resources/translations/wb_it.ts:
```diff
@@ -1032,101 +1033,103 @@ This can be caused by duplicate vertices in your mesh. Try to open your model in
 Restore initial state of the simulation. (%1+Shift+F)</source>
-        <translation type="unfinished"></translation>
+        <translation>Resetta Simulazione.
+Restora lo stato iniziale della simulazione. (%1+Shift+F)</translation>
```
```suggestion
Ripristina lo stato iniziale della simulazione. (%1+Shift+F)</translation>
```

stefaniapedrazzi commented on resources/translations/wb_it.ts:
```diff
@@ -1032,101 +1033,103 @@ This can be caused by duplicate vertices in your mesh. Try to open your model in
     <message>
         <source>Reset Simulation</source>
-        <translation type="unfinished"></translation>
+        <translation>Resetta Simulazione</translation>
```
I would rather use "azzera" than "resetta" (that is more a slang)
```suggestion
        <translation>Azzera Simulazione</translation>
```

stefaniapedrazzi commented on resources/translations/wb_it.ts:
```diff
@@ -713,310 +714,310 @@ This can be caused by duplicate vertices in your mesh. Try to open your model in
     <message>
         <source>Hide Rendering to gain better performance. (%1+4)</source>
-        <translation type="unfinished"></translation>
+        <translation>Nascondi il Rendering per ottenere migliori performance. (%1+4)</translation>
```
```suggestion
        <translation>Nascondi il Rendering per ottenere migliori prestazioni. (%1+4)</translation>
```

stefaniapedrazzi commented on resources/translations/wb_it.ts:
```diff
@@ -713,310 +714,310 @@ This can be caused by duplicate vertices in your mesh. Try to open your model in
     <message>
         <source>Pause the simulation. (%1+0)</source>
-        <translation type="unfinished"></translation>
+        <translation>Pausa la simulazione. (%1+0)</translation>
```
```suggestion
        <translation>Metti in pausa la simulazione. (%1+0)</translation>
```

stefaniapedrazzi commented on resources/translations/wb_it.ts:
```diff
@@ -713,310 +714,310 @@ This can be caused by duplicate vertices in your mesh. Try to open your model in
     <message>
         <source>Clears the Console.</source>
-        <translation type="unfinished"></translation>
+        <translation>Pulisce la Console.</translation>
```
```suggestion
        <translation>Pulisci la Console.</translation>
```

stefaniapedrazzi commented on resources/translations/wb_it.ts:
```diff
@@ -713,310 +714,310 @@ This can be caused by duplicate vertices in your mesh. Try to open your model in
     <message>
         <source>Invalid &apos;fieldOfView&apos; changed to 0.7854. The field of view is limited to pi if the &apos;spherical&apos; field is FALSE.</source>
-        <translation type="unfinished"></translation>
+        <translation>Valore invalido di &apos;fieldOfView&apos; cambiato a 0.7854. Il campo visivo è limitato a pi se il campo &apos;spherical&apos;è FALSO.</translation>
```
```suggestion
        <translation>Valore invalido di &apos;fieldOfView&apos; cambiato a 0.7854. Il campo visivo è limitato a pi se il campo &apos;spherical&apos; è FALSO.</translation>
```

stefaniapedrazzi commented on resources/translations/wb_it.ts:
```diff
@@ -586,28 +586,29 @@ Invalid USE nodes that refer to DEF nodes defined outside the %2 node are turned
     <message>
         <source>&apos;%1&apos; expected to be a file but is a directory.</source>
-        <translation type="unfinished"></translation>
+        <translation>&apos;%1&apos; previsto essere un file ma è una directory.</translation>
```
```suggestion
        <translation>&apos;%1&apos; previsto essere un file ma è una cartella.</translation>
```

stefaniapedrazzi commented on resources/translations/wb_it.ts:
```diff
@@ -586,28 +586,29 @@ Invalid USE nodes that refer to DEF nodes defined outside the %2 node are turned
     <message>
         <source>Expected VRML97 type but found &apos;%2&apos;</source>
-        <translation type="unfinished"></translation>
+        <translation>Previsto un tipo VRML97 ma trovato &apos;%2&apos;</translation>
```
```suggestion
        <translation>Previsto formato VRML97 ma trovato &apos;%2&apos;</translation>
```

stefaniapedrazzi commented on resources/translations/wb_it.ts:
```diff
@@ -586,28 +586,29 @@ Invalid USE nodes that refer to DEF nodes defined outside the %2 node are turned
     <message>
         <source>First item of &apos;%1&apos; field expected to be a file but is a directory.</source>
-        <translation type="unfinished"></translation>
+        <translation>Il primo elemento del campo &apos;%1&apos; è supposto essere un file ma è invece una directory.</translation>
```
```suggestion
        <translation>Il primo elemento del campo &apos;%1&apos; deve essere un file ma è invece una cartella.</translation>
```

stefaniapedrazzi commented on resources/translations/wb_it.ts:
```diff
@@ -369,75 +369,75 @@
     <message>
         <source>Cannot create a directory in the Webots temporary directory &quot;%1&quot;</source>
-        <translation type="unfinished"></translation>
+        <translation>Impossibile creare una directory nella directory temporanea di Webots &quot;%1&quot;</translation>
```
```suggestion
        <translation>Impossibile creare una cartella nella cartella temporanea di Webots &quot;%1&quot;</translation>
```

stefaniapedrazzi commented on resources/translations/wb_it.ts:
```diff
@@ -369,75 +369,75 @@
     <message>
         <source>Cannot create the Webots temporary directory &quot;%1&quot;</source>
-        <translation type="unfinished"></translation>
+        <translation>Impossibile creare la directory temporanea Webots &quot;%1&quot;</translation>
```
```suggestion
        <translation>Impossibile creare la cartella temporanea Webots &quot;%1&quot;</translation>
```

stefaniapedrazzi commented on resources/translations/wb_it.ts:
```diff
@@ -369,75 +369,75 @@
     <message>
         <source>%1 node cannot be inserted at the top level of the node hierarchy.</source>
-        <translation type="unfinished"></translation>
+        <translation>il nodo %1 non può essere inserito al livello superiore della gerarchia dei nodi.</translation>
```
```suggestion
        <translation>Il nodo %1 non può essere inserito al livello superiore della gerarchia dei nodi.</translation>
```

stefaniapedrazzi commented on resources/translations/wb_it.ts:
```diff
@@ -289,67 +289,67 @@
     <message>
         <source>Invalid token &quot;%1&quot;</source>
-        <translation type="unfinished"></translation>
+        <translation>Token &quot;%1&quot; invalido</translation>
```
```suggestion
        <translation>Simbolo &quot;%1&quot; invalido</translation>
```

stefaniapedrazzi commented on resources/translations/wb_it.ts:
```diff
@@ -289,67 +289,67 @@
     <message>
         <source>&apos;%1&apos;: This file was created by Webots %2 while you are using Webots %3. Forward compatibility may not work.</source>
-        <translation type="unfinished"></translation>
+        <translation>Il file &apos;%1&apos; è stato creato da Webots %2 e stai usando Webots %3. Compatibilità con versioni successive potrebbe non funzionare.</translation>
```
```suggestion
        <translation>Il file &apos;%1&apos; è stato creato da Webots %2 e stai usando Webots %3. La compatibilità con versioni successive potrebbe non funzionare.</translation>
```

stefaniapedrazzi commented on resources/translations/wb_it.ts:
```diff
@@ -289,67 +289,67 @@
     <message>
         <source>&apos;%1&apos;: error: Missing header.</source>
-        <translation type="unfinished"></translation>
+        <translation>errore &apos;%1&apos;: Intestazione mancante.</translation>
```
```suggestion
        <translation>&apos;%1&apos;: errore : Intestazione mancante.</translation>
```

stefaniapedrazzi commented on resources/translations/wb_it.ts:
```diff
@@ -289,67 +289,67 @@
     <message>
         <source>&apos;IS&apos; keyword is only allowed in .proto files</source>
-        <translation type="unfinished"></translation>
+        <translation>la parola chiave &apos;IS&apos; è permessa solo in files .proto</translation>
```
```suggestion
        <translation>la parola chiave &apos;IS&apos; è permessa solo in file .proto</translation>
```

stefaniapedrazzi commented on resources/translations/wb_it.ts:
```diff
@@ -229,39 +229,39 @@
     <message>
         <source>End of file reached while a token is expected</source>
-        <translation type="unfinished"></translation>
+        <translation>Fine del file raggiunta sebbene un token sia previsto</translation>
```
```suggestion
        <translation>Fine del file raggiunta sebbene un simbolo sia previsto</translation>
```

stefaniapedrazzi commented on resources/translations/wb_it.ts:
```diff
@@ -109,75 +109,75 @@
     <message>
         <source>reset</source>
-        <translation type="unfinished"></translation>
+        <translation>resetta</translation>
```
```suggestion
        <translation>ripristina</translation>
```

stefaniapedrazzi commented on resources/translations/wb_it.ts:
```diff
@@ -1032,101 +1033,103 @@ This can be caused by duplicate vertices in your mesh. Try to open your model in
         <source>Reset Simulation.
 Restore initial state of the simulation. (%1+Shift+F)</source>
-        <translation type="unfinished"></translation>
+        <translation>Resetta Simulazione.
```
```suggestion
        <translation>Azzera Simulazione.
```

stefaniapedrazzi commented on resources/translations/wb_it.ts:
```diff
@@ -1891,11 +1897,11 @@ The path to this Webots project contains non 8-bit characters. Webots won&apos;t
     <message>
         <source>Reset</source>
-        <translation type="unfinished"></translation>
+        <translation>Resetta</translation>
```
```suggestion
        <translation>Azzera</translation>
```

stefaniapedrazzi reviewed (changes requested):
Thank you very much for your contribution!
It is never easy to translate.
I reported some minor comments and some word changes to try to use italian words instead of slang or "italianized" versions of english words.

The major issue to me seems to be the use of the `&amp;` sign (resulting in the underlined letter in the Webots menu /menu items). This is used as shortcut to open the menu from the keyboard by pressing ``<ALT+<letter>``. So even in the translation we should pay attention that the same letter is not used twice otherwise the the second one is useless and it is even better to not set it.
Could you please review all the menus in the italian version and check that these shortcuts are correctly set?

ad-daniel commented:
Thank you for the feedback, I've incorporated the suggested changes.
With regards to the accelerators, that makes a lot more sense, I misunderstood their function entirely. I've fixed it so there's no doubles.

PS: the source language might require some work in that direction as well, for instance in the File menu there's 4 "O".

stefaniapedrazzi commented on resources/translations/wb_it.ts:
```diff
@@ -196,75 +196,75 @@
     <message>
         <source>edit</source>
-        <translation type="unfinished"></translation>
+        <translation>edita</translation>
```
```suggestion
        <translation>modifica</translation>
```

stefaniapedrazzi reviewed (changes requested):
Thank you!
There are still a couple of duplicated accelerators in "File" and "Modifica" menus, but it is fine like this.

I just have one last changes and then it's all fine for me.

ad-daniel commented:
Thank you, for some reason I didn't consider the grayed out ones, should be all unique now (at least the italian ones). I also made the suggested change and marked it here as resolved but somehow it still requires it.
As for the failed test, it's the same issue that was in the master branch yesterday related to copyright dates

stefaniapedrazzi reviewed (approved):
Thank you!

Is your branch up-to-date with the `webots:master` branch?
